root.sh

```
@@ -50,11 +49,20 @@ prefer_system_check: |
   for FEATURE in $FEATURES; do
       root-config --has-$FEATURE | grep -q yes || { echo "$FEATURE missing"; exit 1; }
   done
+incremental_recipe: |
+  #!/bin/bash -e
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles
+  rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+  cd $INSTALLROOT/test || exit 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)